extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/HudiCatalogRangerSparkExtensionSuite.scala

```diff
@@ -326,6 +326,24 @@ class HudiCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite {
     }
   }
 
+  test("CompactionHoodiePathCommand / CompactionShowHoodiePathCommand") {
+    val path = "hdfs://demo/test/hudi/path"
```

Comment:
file:? relative path?

Reply:
> file:? relative path?

Add a case, hmm, but seems hudi won't format the path like spark.